Sources/NIO/Selector.swift

```diff
@@ -549,7 +549,7 @@ final class Selector<R: Registration> {
                 // Consume event
                 var val: UInt = 0
                 // We are not interested in the result
-                _ = Glibc.read(self.timerFD, &val, MemoryLayout<UInt>.size)
+                _ = Glibc.read(self.timerFD, &val, 8)
```

Comment:
instead of a hard-coded `8` we should use `MemoryLayout.size(ofValue: val)` here.

Reply:
Ah yes, we're writing to `&val` indeed.